### DISCUSSION TITLE
Ошибка после обновления MAXSITE с 106 до 107 версии.

### DISCUSSION THREAD

**Discussion by Talyasik** [maxsite/cms#423] [General]
Здравствуйте.Помогите пожалуйста разобраться где искать возможность исправить данные ошибки
A PHP Error was encountered
Severity: Notice

Message: Undefined index: default_lang

Filename: shjs/index.php

Line Number: 146

A PHP Error was encountered
Severity: Notice

Message: Undefined index: default_lang

Filename: shjs/index.php

Line Number: 147
Эти сообщения висят на всех страницаз сайта.Появились после обновления CMS с 106 до 107 версии.
Заранее спасибо за помощь.
С уважением.

**Comment by maxsite** [comment]
Попробуйте зайти в настройки плагина и нажать ещё раз кнопку сохранить.

**Comment by Talyasik** [comment]
Помогло.Спасибо большое.
